.github/workflows/test.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         with:
           container-name: mir-semantics-ci-${{ github.sha }}
       - name: 'Build stable-mir-json and kmir'
-        run: docker exec --user github-user mir-semantics-ci-${GITHUB_SHA} make stable-mir-json build
+        run: docker exec --user github-user mir-semantics-ci-${GITHUB_SHA} make build
```

Comment:
I actually wouldn't make this change, so that we can have the testing phase just be purely testing. Not worth blocking the PR over.